components/CruResource.vue

```diff
@@ -257,6 +316,10 @@ export default {
       this.$refs.save.clicked();
     },
 
+    handleStepChange(e) {
```

Comment:
given the changes to the wizard slot/templates  i don't think this is needed anymore (along with the local copy of `activeStep` - when used in a template that should come from a slot)